letsdrill.rb

```diff
@@ -1,24 +1,37 @@
 def get_letter_grade(integer)
- 
+
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
+  when 80..89
+    return "B"
+  when 70..79
+    return "C"
+  when 60..69
+    return "D"
+  else
+    return "F"
+  end
 
 end
 
 def shortest_string(array)
 
   #Put your code here!
+  array.min { |a, b|  a.size <=> b.size  }
```

Comment:
Check out [Enumerable#min_by](http://ruby-doc.org/core-2.0.0/Enumerable.html#method-i-min_by) for a way to shorten and make this slightly clearer.